doc/whats-new.rst

```diff
@@ -35,6 +35,10 @@ New Features
 ~~~~~~~~~~~~
 - Performance improvement when constructing DataArrays. Significantly speeds up repr for Datasets with large number of variables.
   By `Deepak Cherian <https://github.com/dcherian>`_
+- Decode more CF attributes on file read (put the referenced variables
```

Comment:
Let's move this to "breaking changes" and add a little more detail. Maybe something like

```
Variables referred to in attributes like `bounds` and `grid_mapping` are now automatically set as coordinate variables. These attributes are moved to :py:attr:`DataArray.encoding` from :py:attr:`DataArray.attrs`. This behaviour is controlled by the `decode_coords` kwarg to :py:func:`open_dataset` and :py:func:`open_mfdataset`
```

Reply:
Moved to breaking changes.